### ISSUE TITLE
Feature - Logging

### ISSUE THREAD

**Issue by jackd248** [jackd248/db-sync-tool#6]
Enable an option to log the synchronization result to a separated file.

**Comment by jackd248** [comment]
Requirements:
- Start of script execution
- Which config file (Sync mode)
- Success (Dump size, dump location) 
- (Errors)

Configurable via "log"-entry in config file.

**Comment by jackd248** [comment]
Example log output for now:
```
2020-09-24 08:48:55,452 - INFO - Starting db_sync_tool
2020-09-24 08:48:55,452 - INFO - Configuration: /path/host.json
2020-09-24 08:49:18,990 - INFO - Database dump file is saved to: /path/_dump_name.sql.tar.gz
2020-09-24 08:49:18,991 - INFO - Successfully created database dump
```
Log will be written if a `log_file` entry is set in the `host.json`

**Comment by jackd248** [comment]
Add verbose feature to log all console output.

**Comment by jackd248** [comment]
Will be released with version **1.7.0**
